tpcx_bb/queries/q10/tpcx_bb_query_10_sql.py

```diff
@@ -88,12 +88,12 @@ def main(data_dir, client, bc, config):
     # These files come from the official TPCx-BB kit
     # We extracted them from bigbenchqueriesmr.jar
     # Need to pass the absolute path for these txt files
-    sentiment_dir = "/".join(config["data_dir"].split("/")[:-3] + ["sentiment_files"])
+    sentiment_dir = os.path.join(config["data_dir"], "sentiment_files")
```

Comment:
Just want to make sure this is intentional as we often save files at 2 directory above this.  

So if the path is  `/raid/tpcx-bb/sf1000/parquet_2gb` the sentiment files were previously placed at ``/raid/tpcx-bb/` . This now assumes the same path as `data_dir` which is better but just want to be sure this is intentional.

Reply:
This change was intentional (see PR description)